documents/decimal.go

```diff
@@ -117,14 +128,25 @@ func (d *Decimal) Bytes() (decimal []byte, err error) {
 
 // SetBytes parse the bytes to Decimal.
 func (d *Decimal) SetBytes(dec []byte) error {
-	if len(dec) < minDecimalByteLength {
+	if len(dec) < minDecimalByteLength || len(dec) > maxDecimalByteLength {
```

Comment:
if the decimal length is less than the minimum, cant we append 0s or we want to enforce always the minimum?

Reply:
we want to enforce the minimum always.